Fix-Website/Templates/info.html

```diff
@@ -0,0 +1,24 @@
+﻿<section class="Info">
+    <div ng-show="checkFrame(1)">
+        <h1>Google Cloud Powershell</h1>
+        <p>Blurb about powershell</p>
+    </div>
+    <div ng-show="checkFrame(2)">
+        <h1>Google Cloud Platform DNS</h1>
+        <table>
+            <tr>
+                <td>Get-GcdChange</td>
+                <td>Lists the ManagedZone's Change resources.</td>
+            </tr>
+        </table>
+    </div>
+    <div ng-show="checkFrame(3)">
```

Comment:
Where is `checkFrame` defined? Try to use an explicit controller reference, as it makes it clearer what is going on. e.g.

````
<div ng-controller="WidgetCtrl as widgetCtrl">
  <h1>{{ widgetCtrl.title }}</h1>
</div>
````